runtime/compiler/CMakeLists.txt

```diff
@@ -115,6 +115,28 @@ if(JITSERVER_SUPPORT)
 	)
 
 	add_custom_target(j9jit_proto DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.h ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.cpp)
+
+	if(JITSERVER_ENABLE_SSL)
```

Comment:
How does `JITSERVER_ENABLE_SSL` get set?

Reply:
At the moment, it's not set. Before the PR Jenkins build could support `--enable-jitserver ` + `--with-openssl=fetched`, JITServer is built against with the system installed OpenSSL. The CI build machines might not have the compatible OpenSSL version installed. The PR build will likely fail for JITServer builds. Therefore,   `JITSERVER_ENABLE_SSL` is not set or enabled yet. 

Once the PR Jenkins build support is ready, we would do the following to enable it as the first step. and the second step is to remove the `JITSERVER_ENABLE_SSL` completely and build OpenSSL in JITServer by default. 
```
ifneq ($(JITSERVER_SUPPORT),)
    export JITSERVER_ENABLE_SSL=1
endif
```

Reply:
Independent of the combination of configure options used in PR builds, this should do as the user requested: you'll need to arrange that. It seems like we need something like a configure option `--enable-jitserver-ssl` that could be added when Jenkins build machines are ready.

Reply:
OpenSSL should be built into JITServer by default. It’s not intended to be a user choice.
It’s only disabled temporarily because initially the PR build on the `jitaas` branch failed due to incompatible libssl is installed on some of the build machines. 
To unblock the PR builds on the `jitaas` branch, `JITSERVER_ENABLE_SSL` was added to comment out all OpenSSL related code.  The code remained as disabled when JITServer is merged into the `master` branch. 

Now this change is just taking steps to eventually remove `JITSERVER_ENABLE_SSL` in JITServer code. In a way, building OpenSSL into JITServer is sort of independent from the flags set at the JVM build level: 
- If no flag is set at the JVM build level, JITServer still builds against the libssl installed on the build system.
- If the flags are set at the JVM build level, JITServer will take them into consideration.

Reply:
If the tests of `JITSERVER_ENABLE_SSL` are to be temporary, they deserve comments to that effect.

Reply:
Will add comments about `JITSERVER_ENABLE_SSL`  in `CMakeLists.txt` and `build/toolcfg/gnu/common.mk`.